packages/app-mobile/tools/buildInjectedJs.ts

```diff
@@ -47,28 +41,43 @@ class BundledFile {
 		this.bundleMinifiedPath = `${this.rootFileDirectory}/${this.bundleBaseName}.bundle.min.js`;
 	}
 
-	private getRollupOptions(): [RollupOptions, OutputOptions] {
-		const rollupInputOptions: RollupOptions = {
-			input: this.sourceFilePath,
-			plugins: [
-				typescript({
-					// Exclude all .js files. Rollup will attempt to import a .js
-					// file if both a .ts and .js file are present, conflicting
-					// with our build setup. See
-					// https://discourse.joplinapp.org/t/importing-a-ts-file-from-a-rollup-bundled-ts-file/
-					exclude: `${this.rootFileDirectory}/**/*.js`,
-				}),
-				nodeResolve(),
-			],
-		};
-
-		const rollupOutputOptions: OutputOptions = {
-			format: 'iife',
-			name: this.bundleName,
-			file: this.bundleOutputPath,
+	private getWebpackOptions(mode: 'production' | 'development'): webpack.Configuration {
+		const config: webpack.Configuration = {
+			mode,
+			entry: this.sourceFilePath,
+			output: {
+				path: this.rootFileDirectory,
+				filename: `${this.bundleBaseName}.bundle.js`,
+
+				library: {
+					type: 'window',
+					name: this.bundleName,
+				},
+			},
+			// See https://webpack.js.org/guides/typescript/
+			module: {
+				rules: [
+					{
+						// Include .tsx to include react components
+						test: /\.tsx?$/,
+						use: 'ts-loader',
+						exclude: /node_modules/,
+					},
+				],
+			},
+			// Increase the minimum size required
+			// to trigger warnings.
+			// See https://stackoverflow.com/a/53517149/17055750
+			performance: {
+				maxAssetSize: 2_000_000, // 2-ish MiB
```

Comment:
CodeMirror needs extra space (this `maxAssetSize` is computed **before** minifying).